version.tf

```diff
@@ -15,36 +15,36 @@
  */
 
 terraform {
-  required_version = ">= 0.12.0"
+  required_version = ">= 0.13.0"
   required_providers {
     # source is not supported in TF12
```

Comment:
nit: remove comment as we are now using 0.13

Reply:
Done